foundation-extensions/mobile-core/signals/README.md

```diff
@@ -37,8 +37,38 @@ import com.adobe.marketing.mobile.*;
 ```
 {% endtab %}
 
-{% tab title="iOS" %}
-​ Add the [Mobile Core](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core) extension to your project using Cocoapods.
+{% tab title="iOS (AEP 3.x)" %}
+​Add the [Mobile Core](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core) extension to your project using Cocoapods.
+
+Add following pods in your `Podfile`:
+
+```text
+pod 'AEPCore'
+pod 'AEPServices'
+pod 'AEPRulesEngine'
+pod 'AEPSignal'
+```
+
+Import the Signal libraries:
+
+#### Swift
+
+```text
+import AEPCore
+import AEPSignal
+```
+
+#### Objective-C
+
+```text
+@import AEPCore;
+@import AEPSignal;
+```
+
+{% endtab %}
+
+{% tab title="iOS (ACP 2.x)" %}
+​Add the [Mobile Core](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core) extension to your project using Cocoapods.
```

Comment:
The Signal extension is included in the Mobile Core extension. Add the [Mobile Core](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core) extension to your project using Cocoapods.